components/filter-function/Filter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import * as Styled from "./Filter.styled";
+import { ClientInterface } from "../../types/form-interfaces";
+
+export interface FilterProps {
+  setActiveFilter?: (filter: string) => void;
+  clients: ClientInterface[];
+}
+
+const Filter = ({ clients, setActiveFilter }: FilterProps) => {
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setActiveFilter(event.target.value);
+  };
+
+  return (
+    <Styled.FilterSelectGroup name="clients" onChange={handleChange}>
```

Comment:
I'd say the naming could be like this as well:

`<Styled.FilterSelect> and
<Styled.FilterOption>`

No need to change I think